ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/DiskResourceMetadataViewImpl.java

```diff
@@ -346,12 +407,17 @@ private String getUniqueAttrName(String attrName, int i) {
     }
 
     private void initUserMdGridEditor() {
-        userGridInlineEditing = new GridInlineEditing<>(userMdGrid);
-        userGridInlineEditing.setClicksToEdit(ClicksToEdit.TWO);
+        userGridRowEditing = new GridRowEditing<>(userMdGrid);
+        userGridRowEditing.setClicksToEdit(ClicksToEdit.TWO);
```

Comment:
Will this still let a user double-click on a row to edit the AVU? What if the user doesn't have write permissions?